framework/web/js/source/jquery.ba-bbq.js

```diff
@@ -1282,96 +1293,6 @@
       timeout_id = setTimeout( poll, $.fn[ str_hashchange ].delay );
     };
     
-    // vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv
-    // vvvvvvvvvvvvvvvvvvv REMOVE IF NOT SUPPORTING IE6/7/8 vvvvvvvvvvvvvvvvvvv
-    // vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv
```

Comment:
This removal does not look relevant to the security fix.

Reply:
I would remove it anyway for the sake of removing amount of code that may need to be fixed/tested for this issue. IMO supporting IE does not make sense anymore in 2024.

Reply:
Agreed. I see this was also suggested upstream: https://github.com/cowboy/jquery-bbq/pull/48/files